{"name": "test_Days[2020-2]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 29 == 31", "trace": "self = <TestYear.Test object at 0x0000028E98273370>, year = 2020, month = 2\n\n    @pytest.mark.parametrize(\"year,month\",[(2022,1),(2022,2),(2020,2),(2022,10)])\n    def test_Days(self,year,month):\n        result = year_month(year,month)\n>       assert result == 31\nE       assert 29 == 31\n\nTestYear.py:28: AssertionError"}, "attachments": [{"name": "stdout", "source": "1107fd20-bc87-4ae1-b780-f323f33562fa-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "year", "value": "2020"}, {"name": "month", "value": "2"}], "start": 1665925791070, "stop": 1665925791070, "uuid": "328bb741-1d38-4ac4-b0d9-6c0360dacf6a", "historyId": "b02ab68fc42ae56c16b94f6552c5ac96", "testCaseId": "bce9ad11c85e3ad0c3ea334c9cd45d98", "fullName": "TestYear.Test#test_Days", "labels": [{"name": "suite", "value": "TestYear"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "DESKTOP-0AC7NBS"}, {"name": "thread", "value": "41744-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestYear"}]}